thrift/storypackage.thrift

```diff
@@ -0,0 +1,10 @@
+
+struct StoryPackageItem {
+  1: required string internalPageCode,
```

Comment:
I don't believe the commas are needed at the end of the line for each field definition.